modules/reference/appvms/business.nix

```diff
@@ -46,130 +47,160 @@ in
-        givc.appvm = {
-          enable = true;
-          name = lib.mkForce "business-vm";
-          applications = [
-            {
-              name = "chromium";
-              command = "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/chromium --enable-features=UseOzonePlatform --ozone-platform=wayland ${config.ghaf.givc.idsExtraArgs}";
-              args = [ "url" ];
-            }
-            {
-              name = "outlook";
-              command = "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/chromium --enable-features=UseOzonePlatform --ozone-platform=wayland --app=https://outlook.office.com/mail/ ${config.ghaf.givc.idsExtraArgs}";
-            }
-            {
-              name = "office";
-              command = "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/chromium --enable-features=UseOzonePlatform --ozone-platform=wayland --app=https://microsoft365.com ${config.ghaf.givc.idsExtraArgs}";
-            }
-            {
-              name = "teams";
-              command = "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/chromium --enable-features=UseOzonePlatform --ozone-platform=wayland --app=https://teams.microsoft.com ${config.ghaf.givc.idsExtraArgs}";
-            }
-            {
-              name = "gpclient";
-              command = "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/gpclient -platform wayland";
-            }
-            {
-              name = "gnome-text-editor";
-              command = "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/gnome-text-editor";
-            }
-            {
-              name = "losslesscut";
-              command = "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/losslesscut --enable-features=UseOzonePlatform --ozone-platform=wayland";
-            }
-          ];
+        microvm = {
+          qemu.extraArgs = lib.optionals (
+            config.ghaf.hardware.usb.internal.enable
+            && (lib.hasAttr "cam0" config.ghaf.hardware.usb.internal.qemuExtraArgs)
+          ) config.ghaf.hardware.usb.internal.qemuExtraArgs.cam0;
+          devices = [ ];
         };
 
-        reference = {
-          programs.chromium.enable = true;
-
-          services.globalprotect = {
+        ghaf = {
+          givc.appvm = {
             enable = true;
-            csdWrapper = "${pkgs.openconnect}/libexec/openconnect/hipreport.sh";
+            name = lib.mkForce "business-vm";
+            applications = [
+              {
+                name = "chromium";
+                command =
+                  "${config.ghaf.givc.appPrefix}/run-waypipe ${config.ghaf.givc.appPrefix}/chromium --enable-features=UseOzonePlatform --ozone-platform=wayland ${config.ghaf.givc.idsExtraArgs}"
+                  + optionalString config.ghaf.givc.enable " --load-extension=${pkgs.open-normal-extension}";
```

Comment:
I think here it is too late to check if givc is enabled

Reply:
That did cross my mind, but eventually decided to have it there anyway, just in case.
Shall I just remove the optionality of the "--load-extension..." -string?

Reply:
Removed optionality of the "--load-extension..." -string